demo/demo-schema/src/main/java/org/apache/servicecomb/demo/CodeFirstRestTemplate.java

```diff
@@ -123,6 +128,21 @@ private void testCodeFirstTextPlain(RestTemplate template, String cseUrlPrefix)
     TestMgr.check(body, result);
   }
 
+  private void testCodeFirstAppXml(RestTemplate template, String cseUrlPrefix) {
+    JAXBPerson person = new JAXBPerson("jake", 22, "it", "60kg");
+    person.setJob(new JAXBJob("developer", "coding"));
+    HttpHeaders headers = new HttpHeaders();
+    headers.add("Accept", MediaType.APPLICATION_XML_VALUE);
+    headers.add("Content-Type", MediaType.APPLICATION_XML_VALUE);
```

Comment:
This demo shows something like we support XML consumer and provider parsing XML contents. But actually they both working in default to APPLICATION/json. Anyway, I'll approve this modification.

Reply:
yes, now servicecomb processing client requests only use application/xml, don't support extend, that means whatever i set in ContentType, it all will be cover to application/xml before send to server, and server will default use json to do decode.
so the xml produceprocessor actually effective in response, when response, provider use xml encode and consumer use xml docode

Reply:
i have change the annocation consumes from xml to json